Screenbox/Pages/PlayerPage.xaml.cs

```diff
@@ -109,7 +109,6 @@ protected override void OnKeyDown(KeyRoutedEventArgs e)
                     VideoView.ContextFlyout.ShowAt(PlayerControls,
                         new FlyoutShowOptions { Placement = GlobalizationHelper.IsRightToLeftLanguage ? FlyoutPlacementMode.TopEdgeAlignedLeft : FlyoutPlacementMode.TopEdgeAlignedRight });
                     break;
-                case VirtualKey.Escape when shouldHideControls:
```

Comment:
Maybe I missed some context. Why do we want to remove this condition?

Reply:
It's been so long, idk what happened here. It may have to do with the Escape key not behaving as expected in a KeyboardAccelerator context.